From 387efb0ac01e40cd85cf9f5ce770be6437265ca4 Mon Sep 17 00:00:00 2001
From: Antoine Martin <dev@ayakael.net>
Date: Fri, 4 Jun 2021 13:32:53 +0000
Subject: [PATCH] FIxed Net40 location

Because paths on Linux are case sensitive. Applications insights looks
for <dotnet/applicationsinsights>/src/Core/Managed/net40 in net40 
while the repo has a Net40. Patch moves whatever is in Net40 to net40

(see https://github.com/microsoft/ApplicationInsights-dotnet/issues/2498)
---
 src/Core/Managed/{Net40 => net40}/CorrelationHelper.cs | 0
 1 file changed, 0 insertions(+), 0 deletions(-)
 rename src/Core/Managed/{Net40 => net40}/CorrelationHelper.cs (100%)

diff --git a/src/Core/Managed/Net40/CorrelationHelper.cs b/src/Core/Managed/net40/CorrelationHelper.cs
similarity index 100%
rename from src/Core/Managed/Net40/CorrelationHelper.cs
rename to src/Core/Managed/net40/CorrelationHelper.cs
-- 
2.30.2

